jasmine/src/inverted-index.js

```diff
@@ -0,0 +1,149 @@
+class Index {
+
+    // Method to create an inverted-index of the documents
+    createIndex(filePath) {
+
+        // Here, self refers to the Index object
+        var self = this;
+
+        // Initialise Object to store the inverted-index
+        this.indexObj = {};
+
+        /* Fetch the document 
+         * Resolve the promise and store the data in a variable, 'finallySomeData'
+         */
+        return fetch(filePath).then(function(response) {
+            return response.json();
+
+        })
+
+        .then(function(finallySomeData) {
+            self.books = finallySomeData;
+
+            // Call the populateIndex function to create the inverted-index
+            populateIndex();
+
+        });
+
+        function populateIndex() {
+            // Iterate through the data obtaining each book's position in the document.
+            self.books.forEach((book, docIndex) => {
```

Comment:
**Code Quality - Nitpick**

I see you have used an arrow function here and on line 42 but not on line 15. Try chose one over the other and use it consistently.